services/testing-framework/claudie_test.go

```diff
@@ -32,9 +35,48 @@ const (
 	maxTimeoutSave = 60 * 12 // max allowed time for config to be found in the database
 )
 
+var (
+	// get env var from runtime directly so we do not pollute original envs package by unnecessary variables
+	cleanUpFlag = os.Getenv("AUTO_CLEAN_UP")
```

Comment:
my only suggestion here would be to name the thing `autoCleanUpFlag`, so that it's clearer.